### ISSUE TITLE
Support `baseUrl` resolution

### ISSUE THREAD

**Issue by fsmaia** [simonhaenisch/rollup-plugin-typescript-paths#12]
Not every resolution is based on `paths`, like in this scenario:

tsconfig.json:
```json
{
  "compilerOptions": {
    "baseUrl": "./src"
  }
}
```

src/index.ts:
```ts
import { example } from 'modules/Example';
```

src/modules/Example/index.ts:
```ts
export const example = () => {...};
```

This resolution can be handled with `moduleResolveName`, natively.

**Comment by simonhaenisch** [comment]
ok cool thanks for the issue... i'll try to get to it this week, or you can make a PR if you want (:

**Comment by simonhaenisch** [comment]
See PR, not sure that's a good solution though because I don't think it's backwards compatible. Might need to either do a major bump, or hide it behind an option so it's opt-in behavior.

**Comment by fsmaia** [comment]
I liked the PR!!

Yep, compatibility is going to be an issue!  Maybe that's why most typescript paths libs and plugins use the same strategy as this repository.

**Comment by simonhaenisch** [comment]
Yeah I see the use case for this, however it's also kinda out-of-scope for this plugin because it's supposed to solely deal with path aliases. However having a separate plugin for this is also kinda overkill 🤷🏻‍♂️

I'll change the PR to turn it into an opt-in behavior.

**Comment by simonhaenisch** [comment]
I just released version `1.4.0`, can you please try it out?

You'll have to set `nonRelative: true` in the plugin's options:

```ts
import { typescriptPaths } from 'rollup-plugin-typescript-paths';

export default {
  // ...
  plugins: [typescriptPaths({ nonRelative: true })],
};
```

**Comment by fsmaia** [comment]
It worked for me! Huge thanks!
